docs/UserGuide.md

```diff
@@ -101,23 +102,25 @@ pay attention to. </div>
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:** used to highlight dangers. </div>
```

Comment:
We can potentially change the color of this to something that pops out a bit more.